### PR TITLE
vm: improve `runTx` and `evm` debug logs

### PR DESCRIPTION
**Pull request by ryanio**
This PR improves the runTx debug logs as investigated in #1446 and https://github.com/ethereumjs/ethereumjs-monorepo/issues/1446#issuecomment-1026463341

### CODE REVIEW

**Review by acolytec3 — APPROVED** [review]
LGTM!  Not sure why the code coverage is complaining but this shouldn't need new test coverage since the changes are limited to debug messages.  @holgerd77 thoughts on whether we could admin merge this given above?

**Comment by acolytec3** [comment]
image

Was able to find a test where the RETURN opcode is used and it populates that `execResult.returnValue` field.  It's all 0s but at least we can see it being populated in the revised logger message.  LGTM!

**Review by acolytec3 — DISMISSED** [review]
LGTM!

File: packages/vm/src/runTx.ts [hunk 1]
```diff
@@ -395,22 +395,19 @@ async function _runTx(this: VM, opts: RunTxOpts): Promise<RunTxResult> {
...
     debug(
-      `Received tx results gasUsed=${results.gasUsed} execResult: [ gasUsed=${
-        results.gasUsed
-      } exceptionError=${
-        results.execResult.exceptionError ? results.execResult.exceptionError.error : ''
-      } returnValue=${short(results.execResult.returnValue)} gasRefund=${
-        results.execResult.gasRefund
-      } ]`
+      `Received tx execResult: [ gasUsed=${gasUsed} exceptionError=${
+        exceptionError ? `'${exceptionError.error}'` : 'none'
```
Comment (acolytec3) [inline packages/vm/src/runTx.ts:409]: What are the `'...'` for in the exceptionError?  Is it just to clearly demarcate the error message?

**Review by acolytec3 — APPROVED** [review]

File: packages/vm/src/runTx.ts (hunk 1 above)
Comment (ryanio) [reply packages/vm/src/runTx.ts:409]: What do you mean? I don't see that. Maybe you are confusing with the short() formatting on returnValue that after a certain length adds ...

Comment (ryanio) [reply packages/vm/src/runTx.ts:409]: Ohh the single quotes? To contain the error message so it looks like `exceptionError='out of gas'`

**Comment by ryanio** [comment]
thanks!!

File: packages/vm/src/runTx.ts (hunk 1 above)
Comment (acolytec3) [reply packages/vm/src/runTx.ts:409]: Yup, sorry, that comment wasn't clear but thanks for the clarification!

**Pull request merged**